enclave/core/malloc.c

```diff
@@ -32,6 +32,9 @@
 
 #endif
 
+/* If true, disable the debug malloc checking */
+bool oe_disable_debug_malloc_check;
```

Comment:
Is this part of this PR?

Reply:
The declaration was put in the enclave/core/calls.c, which is removed in this PR. I find moving to this file is fine (otherwise we will have a file that does only this variable declaration.). Do you have any suggestion?

Reply:
Ah Ok. I didn't realize it was a preexiting variable that is being moved.
This is the right place for the variable.